code/__DEFINES/~skyrat_defines/clockworkcult.dm

```diff
@@ -0,0 +1,180 @@
+GLOBAL_LIST_EMPTY(city_of_cogs_spawns) //Anyone entering the City of Cogs spawns here
+GLOBAL_LIST_EMPTY(servant_spawns) //Servants of Ratvar spawn here
+
+#define PRESSURE_PLATE_LAYER 2.49
+#define CHANNEL_JUSTICAR_ARK 1020
+#define NOJAUNT_1					(1<<0)
+#define ZTRAIT_REEBE "Reebe"
+#define ZTRAITS_REEBE list(ZTRAIT_REEBE = TRUE, ZTRAIT_BOMBCAP_MULTIPLIER = 0.5)
+#define ROLE_SERVANT_OF_RATVAR	"Servant of Ratvar"
+#define ANTAG_HUD_CLOCKWORK		22
+#define TRAIT_STARGAZED			"stargazed"	//Affected by a stargazer
+#define STARGAZER_TRAIT "stargazer"
+#define CLOCKCULT_SERVANTS 4
+#define COMSIG_ATOM_RATVAR_ACT "atom_ratvar_act"
+#define LIGHT_COLOR_CLOCKWORK 	"#BE8700"
+#define TELEPORT_MODE_CLOCKWORK 2
+#define COMSIG_ATOM_EMINENCE_ACT "atom_eminence_act"
+#define COMSIG_CLOCKWORK_SIGNAL_RECEIVED "clock_received"			//! When anything the trap is attatched to is triggered
+#define VANGUARD_TRAIT "vanguard"
+#define INCORPOREAL_MOVE_EMINENCE 4 //! same as jaunt, but lets eminence pass clockwalls
+#define STATUS_EFFECT_INTERDICTION /datum/status_effect/interdiction //! The affected is inside the range of an interdiction lens
+#define MOVESPEED_ID_INTERDICTION						"interdiction"
+
+#define ismovableatom(A) ismovable(A)
```

Comment:
Seems a but redundant?

Reply:
necessary for how its handled, the status effect itself doesnt actually have a speed defined, nor does it have a movespeed ID, which appears to be necessary; looking at other examples, and beestations example.

This is also used to display a HUD element, so the double define is necessary as they do separate things.

Reply:
Yeah but can't you just do ismovable instead of isatommovable?

Reply:
i dont know probably